README.md

```diff
@@ -38,12 +38,14 @@ the demo requires the [React Native CLI](https://facebook.github.io/react-native
 $ npm install -g react-native-cli # install victory-native if you haven't already
```

Comment:
NIT -- not part of your PR, but if already changing this file can you pick up:

```
# install react-native-cli if you haven't already
```

?